runtime/compiler/runtime/Runtime.cpp

```diff
@@ -1218,7 +1219,7 @@ void initializeCodeRuntimeHelperTable(J9JITConfig *jitConfig, char isSMP)
    SET(TR_X86interpreterUnresolvedFieldSetterGlue,          (void *)interpreterUnresolvedFieldSetterGlue,          TR_Helper);
    SET(TR_X86interpreterUnresolvedConstantDynamicGlue,      (void *)interpreterUnresolvedConstantDynamicGlue,      TR_Helper);
    SET(TR_X86interpreterStaticAndSpecialGlue,               (void *)interpreterStaticAndSpecialGlue,               TR_Helper);
-
+   SET(TR_X86CRC32C,                                        (void *)crc32c,   TR_Helper);
```

Comment:
`TR_X86java_util_zip_CRC32C_updateBytes` is more descriptive of what you're providing an intrinsic for.